proto-public/pbmulticluster/v2beta1/computed_exported_services.proto

```diff
@@ -11,17 +11,17 @@ import "pbresource/resource.proto";
 message ComputedExportedServices {
   option (hashicorp.consul.resource.spec) = {scope: SCOPE_PARTITION};
 
-  repeated ComputedExportedService consumers = 1;
+  repeated ComputedExportedService services = 1;
 }
 
 message ComputedExportedService {
   hashicorp.consul.resource.Reference target_ref = 1;
-  repeated ComputedExportedServicesConsumer consumers = 2;
+  repeated ComputedExportedServiceConsumer consumers = 2;
 }
 
-message ComputedExportedServicesConsumer {
+message ComputedExportedServiceConsumer {
```

Comment:
This is actually a consumer of a singular exported service, not a consumer for multiple exported services